### PR TITLE
 refactor: change dynamic code to static

### PR DESCRIPTION
**Pull request by NabbeunNabi**
Resolves https://github.com/ourjapanlife/findadoc-web/issues/514

🔧 What changed
The ModDashboardLeftNav was being dynamically generated. But now it is all static with the same logic as before.

📸 Screenshots
Nothing Changed in terms of design. Before and after are the same.

image

### CODE REVIEW

**Comment by theyokohamalife** [comment]
Looks good! Could you add a Cypress test that checks that the elements are visible?

File: components/ModDashboardLeftNavbar.vue [hunk 1]
```diff
@@ -1,64 +1,56 @@
...
-            <button @click="updateSubmissionListViewState(submissionListEntry.value)"
-                class="flex flex-row items-center text-start p-1" :key="submissionListEntry.value"
-                :class="{ 'border-b border-gray-300': index !== submissionListViewOptions.length - 1 }"
-                v-for="(submissionListEntry, index) in submissionListViewOptions ">
-                <div class="flex items-center justify-center w-1/5">
-                    <component :is="submissionListEntry.svgIcon" class="h-5" />
-                </div>
-                {{ $t(submissionListEntry.displayText) }} ({{ submissionListEntry.count }})
-            </button>
+            <div class="flex flex-row w-4/5 justify-start items-center border-b-2 border-slate-200">
```
Comment (ermish) [inline components/ModDashboardLeftNavbar.vue:12]:
```suggestion
            <div class="flex w-4/5 justify-start items-center border-b-2 border-slate-200">
```

can remove redundant `flex-row`

File: components/ModDashboardLeftNavbar.vue [hunk 2]
```diff
@@ -1,64 +1,56 @@
...
+                <SVGNoteStackAddSvg class="h-4" />
+                <button data-testid="mod-dashboard-leftnav-for-review"
+                    @click="updateSubmissionListViewState('forReview')"
+                    class="flex flex-row items-center text-start p-1">
+                    {{ $t("modDashboardLeftNav.forReview") }} ({{ 0 }})
+                </button>
+            </div>
+            <div class="flex flex-row w-4/5 justify-start items-center border-b-2 border-slate-200">
```
Comment (ermish) [inline components/ModDashboardLeftNavbar.vue:20]: ♻️ 
```suggestion
            <div class="flex w-4/5 justify-start items-center border-b-2 border-slate-200">
```

File: components/ModDashboardLeftNavbar.vue [hunk 3]
```diff
@@ -1,64 +1,56 @@
...
+                <SVGCheckBoxSvg class="h-5" />
+                <button data-testid="mod-dashboard-leftnav-approved" @click=" updateSubmissionListViewState('approved')"
+                    class="flex flex-row items-center text-start p-1 ">
+                    {{ $t("modDashboardLeftNav.approved") }} ({{ 0 }})
+                </button>
+            </div>
+            <div class="flex flex-row w-4/5 justify-start items-center ">
```
Comment (ermish) [inline components/ModDashboardLeftNavbar.vue:27]: ♻️ 
```suggestion
            <div class="flex w-4/5 justify-start items-center ">
```

File: components/ModDashboardLeftNavbar.vue [hunk 4]
```diff
@@ -1,64 +1,56 @@
...
-{
-    displayText: "Rejected",
-    value: "rejected",
-    svgIcon: DisabledByDefault,
-    count: 0,
-}]
 
-const selectedSubmissionListView: Ref<String> = ref(submissionListViewOptions[0].value)
-const selectedDashboardView: Ref<String> = ref(modDashboardViewOptions[0].value)
+const selectedSubmissionListView: Ref<string> = ref("forReview")
```
Comment (ermish) [inline components/ModDashboardLeftNavbar.vue:45]: future improvement: can  make these strings an enum for visibility and preventing typos

**Review by ermish — APPROVED** [review]
looks great! 💯

**Pull request merged**